librz/asm/arch/evm/evmdis.c

```diff
@@ -0,0 +1,374 @@
+// SPDX-FileCopyrightText: 2023 gogo <gogo246475@gmail.com>
+// SPDX-FileCopyrightText: 2013-2018 pancake <pancake@nopcode.org>
+// SPDX-License-Identifier: LGPL-3.0-only
+
+#include <rz_types.h>
+#include <rz_util.h>
+#include <rz_asm.h>
+#include <rz_lib.h>
+
+static int evmDisass(RzAsmOp *op, const ut8 *buf, int len) {
+	int instruction_size = 1;
+	char buf_asm[2000];
+	const unsigned char out[256];
+	const unsigned char opcode[256];
+	switch (buf[0]) {
+		case 0x00:
+			rz_str_cpy(buf_asm, "STOP") break;
```

Comment:
There is no need to use `rz_str_cpy` when you have to push data into `RzStrBuf`.
just do `rz_strbuf_set` for static strings and `rz_strbuf_appendf` for adding extra info to the string.